### ISSUE TITLE
Dont loading just Spotify Logo

### ISSUE THREAD

**Issue by MCMARCELO** [skuethe/MMM-Spotify#125]
Hi, i install the module and all worked.
Now it doesnt work anymore, i think since i install a node update. but i downgrade already and even dont work.
It comes just a Spotify logo at all.

Hope you can help me, config look fine and syntax

**Comment by skuethe** [comment]
Hey there. Hard to tell with this few details.
Please verify the installation steps from the [READEME](https://www.github.com/skuethe/MMM-Spotify/tree/master/README.md) - especially that the spotify.config.json exists and has the correct content.

first thought would be that something is missing there.

**Comment by skuethe** [comment]
Did you get this resolved?

**Comment by Moerup** [comment]
I have the same issue.
The Spotify icon briefly pops up on the screen and then goes black.
But if I play via my phone, the window pops up with the correct song.

**Comment by skuethe** [comment]
that does not sound like an issue in it's current state.
This module only shows content if you start playing songs on Spotify.

so in your example: if you start playing a song from your mobile, the module detects this from Spotify API and will display the song which is playing.

if you pause the song the module will continue to display the song for a a few minutes. if you do not un-pause it will the go back into "hiding".

So to sum it up:
the idea is to start Spotify from another device and then be able to control it from this module.

**Comment by Moerup** [comment]
I completely thought it was a dedicated player for the magic mirror..

Sorry about the issue, everything seems to be working as intended then :)

man. 11. jan. 2021 23.55 skrev skuethe <notifications@github.com>:

**Comment by skuethe** [comment]
It more or less is. Once you start playing music from another source (mobile, desktop app, browser, ...), then you have more or less full control of the music.

The problem with integrating a "full" player is, how to handle music selection / playlists / music searches. That is a pretty big chunk of work with a lot of open (design) questions behind it.

I am, however, currently working on optimizing the implementation a bit. So f.e. if "nothing" is playing, that the spotify logo is being displayed all the time and that it also includes the "account switch" button.
Maybe this will also open up some other implementations in the future to something like "resume last played music playlist" option. But this is now just ideas and I haven't looked into it at all.

**Comment by Moerup** [comment]
Sounds interesting, thanks for all the work you have put into this module!

**Comment by skuethe** [comment]
@MCMARCELO if there is anything else I can help you with, get back to me. I will close the issue for now.
